src/card.css.ts

```diff
@@ -470,3 +372,16 @@ export const cardContainer = style({
         clear: 'both',
     },
 });
+
+export const dataCardTopActionsWithoutIcon = style({
+    marginRight: -17,
+    marginTop: -9,
```

Comment:
can you explain this weird values?

Reply:
They were already there, I just moved them to the css file and fixed an off-by-one in the values (https://github.com/Telefonica/mistica-web/blob/master/src/card.tsx#L792).

I will add a comment in the code to explain a bit better.

Reply:
Added comment